<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryWindow</class>
 <widget class="QMainWindow" name="EntryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EntryWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="inputLayout_">
     <item>
      <widget class="QLabel" name="lableInput_">
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputUserName_"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="buttonLayout_">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnOK_">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel_">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
